source/settings/pigeonhole.rst

```diff
@@ -526,6 +526,25 @@ Sieve Plugin Settings
                  matched in those tests.
    ============= ===========================================================
 
+.. pigeonhole:setting:: sieve_max_script_size
+   :plugin: sieve
+   :values: @size
+
+   The maximum size of a Sieve script. Default is 4 megabytes.
```

Comment:
This now exists two times in this here document with different values/descriptions.